test/e2e/adminapi_cluster_list.go

```diff
@@ -85,6 +80,12 @@ func testAdminClustersList(ctx context.Context, path, wantResourceID string) {
 
 		params = nextParams(list.NextLink)
 	}
+	return ocs
+}
+
+func testAdminClustersList(ctx context.Context, path, wantResourceID string) {
+	By("requesting the cluster document via RP admin API")
```

Comment:
Might want to change a message so there is no duplication in output (currently `testAdminClustersList` produces this message and `adminClustersList` and also outputs exactly the same).